specs/olm.spec

```diff
@@ -30,11 +30,13 @@ Steps:
   * Configure GitHub token for git resolver in TektonConfig
   * Configure the bundles resolver
   * Enable console plugin
-  * Validate pipelines deployment
+  * Enable stateful-set in tektonconfig
```

Comment:
We reference installer sets as `installersets` in our codebase so I would do something similar here... `statefulset` (without dash)